TestCases/TestCaseFactory.cs

```diff
@@ -0,0 +1,34 @@
+using System;
+using KBS.TestCases.Contracts;
+using KBS.TestCases.TestCases.RequestResponse;
+using KBS.TestCases.TestCases.Webshop;
+
+namespace KBS.TestCases
+{
+    public enum TestCaseType
```

Comment:
word deze enum alleen hier gebruikt of door de hele solution?

Reply:
Throughout the solution, but always together with the TestCaseFactory.